src/api/app/controllers/source_controller.rb

```diff
@@ -564,7 +564,7 @@ def project_command_undelete
   def project_command_release
     params[:user] = User.session!.login
 
-    @project = Project.get_by_name(params[:project], includeallpackages: 1)
+    @project = Project.get_by_name(params[:project], includeallpackages: true)
```

Comment:
```suggestion
    @project = Project.get_by_name(params[:project], include_all_packages: true)
```
What about to make it more clear the name of the variable?

Reply:
not sure if this makes anything more clear, but why not...

Reply:
Yes, it makes things more clear. Thanks!